src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/PcsVmrBackFlower.cs

```diff
@@ -116,7 +113,7 @@ await _vmrCloneManager.PrepareVmrAsync(
             cancellationToken);
 
         // Prepare repo
-        SourceMapping mapping = _dependencyTracker.GetMapping(mappingName);
+        SourceMapping mapping = _dependencyTracker.GetMapping(subscription.SourceDirectory);
```

Comment:
nit: not sure if we should even do this, but I was searching for `subscription.SourceRepository` to see where we're calling `PrepareVmrAsync` with it, but we're using the `build.GetRepository` uri there. Maybe we should change it a few lines above just to be a bit more consitent

Reply:
I don't follow

Reply:
on line 109, we're calling `PrepareVmrAsync` to make sure we use the correct VMR (dotnet/dotnet, or maestro-auth-test.....), and we need to pass it the vmr URI. We get the uri from the build we're flowing, but we could also get it from the subscription, like we do in Forward Flow. It's a minor thing, but it was just something I was expecting and couldn't find. Not sure if it makes sense

Reply:
it's not in this PR, those changes

Reply:
Ah, right. I wonder which is better - the build or the subscription to get this from?

Reply:
they should always be the same. We can use the subscription both in Forward and Backward flow, so it makes things a bit more consistent between them

Reply:
I agree they should be the same but I am not sure which is better. Don't the builds get their GitHub URI computed in case the SHA is available publicly?

I wonder what we put in V.D.xml when we update dependencies

Reply:
Ah, right, that's true. So I'd leave it to come from subscription in forward and from build in backward flows